pkg/kubelet/volumes.go

```diff
@@ -206,14 +206,14 @@ func (kl *Kubelet) getPodVolumes(podUID types.UID) ([]*volumeTuple, error) {
 }
 
 // cleanerTuple is a union struct to allow separating detaching from the cleaner.
```

Comment:
`allow separating detaching from the cleaner` -> `allow separating detaching from the mounting`